extensions/cornerstone/src/init.tsx

```diff
@@ -155,118 +151,12 @@ export default async function init({
   initWADOImageLoader(userAuthenticationService, appConfig);
 
   /* Measurement Service */
-  const measurementServiceSource = connectToolsToMeasurementService(
+  this.measurementServiceSource = connectToolsToMeasurementService(
```

Comment:
what is `this.`?

Reply:
It is the extension object - this is a really ugly case where a result of one call in the extension is re-used later on in another spot.  I think you set it up so the init was in the commands module previously, but that is just as ugly because it means you delay init till the commands module is setup.